ldap/servers/slapd/entry.c

```diff
@@ -1197,7 +1195,7 @@ str2entry_dupcheck(const char *rawdn, const char *s, int flags, int read_statein
     if (e->e_flags & SLAPI_ENTRY_FLAG_TOMBSTONE) {
         /* tombstone */
         if (_entry_set_tombstone_rdn(e, slapi_entry_get_dn_const(e))) {
-            slapi_log_err(SLAPI_LOG_TRACE, "str2entry_dupcheck",
+            slapi_log_err(SLAPI_LOG_ERR, "str2entry_dupcheck",
```

Comment:
I think that logging the tombstone error at ERR level is OK because:
  - There should be no error unless something is seriously amiss (like a tampered import file) so increasing the log level does not generate noise.
  - Failing to import tombstone may lead to serious replication problems (broken replication or divergent entries among the replicas) so I do not think that such error should be silent.